rodan/jobs/core.py

```diff
@@ -5,8 +5,12 @@
 # import re
 import shutil
 import subprocess
+import sys
 import tempfile
 import time
+import zipfile
+
+from StringIO import StringIO
```

Comment:
This too is problematic between python2/python3, because `StringIO.StringIO` => `io.StringIO`
Use: 
```python
import six
six.StringIO
```